package.json

```diff
@@ -30,7 +34,7 @@
     "test:scss:styleguide:color": "stylelint \"source/sass/**/*.scss\" \"source/js/**/*.scss\" \"!source/sass/**/_colors.scss\" --syntax scss --config .stylelintrc-colors.js",
     "test:css": "stylelint \"network-api/networkapi/{,!(frontend)/**/}*.css\" --syntax scss",
     "test": "run-s test:** build",
-    "watch": "npm run build && run-p watch:**",
+    "watch": "npm run build:dev && run-p watch:**",
```

Comment:
ideally this would be `run-s build:dev watch-all` with a separate task `"watch-all": "run-p watch:*`